pkg/simulate/pushsync/metrics.go

```diff
@@ -0,0 +1,76 @@
+package pushsync
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/push"
+	"github.com/prometheus/common/expfmt"
+)
+
+type metrics struct {
+	uploadedChunks   *prometheus.CounterVec
+	downloadedChunks *prometheus.CounterVec
+	downloadCount    *prometheus.CounterVec
+}
+
+func newMetrics(runID string, pusher *push.Pusher) metrics {
+	namespace := "beekeeper"
```

Comment:
consider converting this to how we do this in `bee`. not 100% this format is more readable